server.js

```diff
@@ -10,23 +9,34 @@ const knex           = require('knex')({
   }
 })
 
+const app = express()
+
 app.use(bodyParser.json())
 app.use(bodyParser.urlencoded({ extended: true}))
-app.use(methodOverride('_method'))
+app.use(express.static('public'))
+
 
+//Store user data in database
 app.post('/signup', function (req, res) {
   knex('users')
-    .insert({name: req.body.name, age: req.body.age, mobile: req.body.num})
+    .insert({
```

Comment:
If this object that you are inserting has the name properties as `req.body`, then you can just insert `req.body`. No need to make a whole new object.